lib/core/widgets/home_page_app_bar.dart

```diff
@@ -174,7 +175,7 @@ class Action extends StatelessWidget {
           );
         } else {
           return Padding(
-            key: const ValueKey("search icon"),
+            key: ValueKey(AppLocalizations.of(context).searchIcon),
```

Comment:
Same as [this](https://github.com/SankethBK/diaryvault/pull/74/files#r1349925553)

Reply:
I didn't change it at first commit. Then in the second commit I updated it. No issues will revert it back and make a new PR

Reply:
don't create new PR it would be difficult to manage, add a commit to this PR only

Reply:
Got it will do.